.genx/configure.js

```diff
@@ -138,4 +138,8 @@ module.exports = async (data, utils) => {
   ].forEach(projectName => {
     utils.rename(path.resolve(serverJvmRoot, projectName), path.resolve(serverJvmRoot, projectName.replace(appNamePlaceholder, data.appName)));
   });
+
+  function move(oldPath, newPath) {
```

Comment:
where is this function called from?

Reply:
Thanks, forgot to push the changes

Reply:
Yes, it's actual move and overwrite option while `rename` fails if the target directory is not empty.